front-end/linux-help.html

```diff
@@ -42,9 +42,10 @@
             </p>
             <p class="code">
                 sudo usermod -a -G dialout $USER
+		sg dialout "npm start"
```

Comment:
It's a cousin of the newgrp(1) tool that logs into a new group. sg(1) let's you run a command as a different group, so you don't really need to log off or reboot in this case.